setup.py

```diff
@@ -205,7 +205,7 @@ def do_setup():
             'flask-login==0.2.11',
             'flask-swagger==0.2.13',
             'flask-wtf==0.12',
-            'funcsigs>=1.0.2, <1.1',
```

Comment:
It's ultimately an issue with funcsigs, I'll open an issue for it.

> We could also require a higher version of setup tools.

Afaict the setuptools requirement was what introduced the funcsigs bug for me; doing the same in Airflow would spread the issue more.